### DISCUSSION TITLE
Welcome to laravel-vue-admin-panel Discussions!

### DISCUSSION THREAD

**Discussion by balajidharma** [balajidharma/laravel-vue-admin-panel#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by codedsultan** [comment]
i like to make some contribution, found some bugs which i fixed

**Comment by NouheDerwiche** [comment]
Hi, I want to ask you about the project laravel-vue-admin-panel. When I want to test any CRUD with Postman, I have this problem with CSR
![probleme CSRF](https://github.com/user-attachments/assets/eb97dbfc-74d2-4887-ad83-e4a70db9e776)
F: "message": "CSRF token mismatch," but I configure the parameter for Postman correctly with cookies and X-XSRF-TOKEN. Help me, please .
